front-end/src/components/MainPage.jsx

```diff
@@ -19,6 +19,42 @@ class MainPage extends Component {
     store.dispatch(openAnalysisPage());
   }
 
+  downloadTranscriptTemplate = () => {
+    const element = document.createElement("a");
+    const data = {"data": [
+        {
+          "": "23606",
+          "project_id": "1",
+          "turn_id": "dd8e6fee-2329-4470-9cc0-0c50f3d40313",
+          "ts": "2022-04-20 18:01:59.863 UTC",
+          "session_id": "c1ade6a3e2d0171d8813a2279f69a97171258a6c341547919fc0a3f95909b744",
+          "platform": "prototype",
+          "interaction_metadata": "{\"end\":false,\"locale\":\"en-US\"}",
+          "trace_format": "trace",
+          "trace_type": "speak",
```

Comment:
Maybe eliminate the items we don't actually use. @helias22  will know for sure which elements those are

Reply:
We only need trace_type, trace_payload and project_id overall. So for this chunk of code we just need trace_type.